### DISCUSSION TITLE
Change icon-element aspect ratio

### DISCUSSION THREAD

**Discussion by kamssiopeia** [davatorium/rofi#2003] [Q&A] [answered]
Hey guys, from what I can tell `element-icon` is square and it's height is automatically adjusted to width. Is there any way to change it? I'm trying to fit 16:9 wallpapers nicely

White background is `element-icon`
![image](https://github.com/davatorium/rofi/assets/22861342/adb62e07-53b0-4c7e-807a-9f50c53822a2)

**Comment by DaveDavenport** [comment, marked as answer]
No this is currently not possible for element-icon.
